src/controllers/libraryController.ts

```diff
@@ -62,9 +63,19 @@ export class LibraryController implements Disposable {
         Settings.updateReferencedLibraries(setting);
     }
 
-    public async removeLibrary(library: string) {
+    public async removeLibrary(removalFsPath: string) {
         const setting = Settings.referencedLibraries();
-        setting.exclude = this.updatePatternArray(setting.exclude, workspace.asRelativePath(library, false));
+        const removedPaths = _.remove(setting.include, (include) => {
+            if (path.isAbsolute(include)) {
+                return Uri.file(include).fsPath === removalFsPath;
+            } else {
+                return include === workspace.asRelativePath(removalFsPath, false);
```

Comment:
Also apply `asRelativePath` on include to ensure they're in the same form?
```suggestion
                return workspace.asRelativePath(include, false) === workspace.asRelativePath(removalFsPath, false);
```